torch/_refs/special/__init__.py

```diff
@@ -21,3 +25,17 @@
     type_promotion_kind=utils.ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT,
     aten_op=torch.ops.aten.special_i1e,
 )
+
+@register_decomposition(torch.ops.aten.logit)
+@out_wrapper
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("self",),
+    type_promotion_kind=utils.ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT,
+)
+def logit(self: Tensor, eps: Optional[float] = None) -> Tensor:
+    if eps is None:
+        eps = -1.0
+    lo = eps
+    hi = 1 - eps
+    self = torch.clamp(self, lo, hi)
+    return (self / (1 - self)).log()
```

Comment:
How do these torch operations work with decomposing to prims? Also we should add an OpInfo for refs.logit.

I think all these references exist except for clamp -- which should be straightforward to add now that @ngimel has fixed its eager behavior